### DISCUSSION TITLE
Super note link editing doesn't work

### DISCUSSION THREAD

**Discussion by DarkArc** [standardnotes/forum#3235] [General]
The new link improvements in super notes are great (particularly how it seems to read links from the clipboard when you do Ctrl + K). However, I can't seem to edit links using the pencil. 

To reproduce, highlight some text, press the link button, type in something like `http://standardnotes.com`, and press enter OR click the checkmark. The dialog will disappear, however, no link destination will be saved.

**Comment by effieeee** [comment]
Hi @DarkArc, I have to confirm if this is a visual bug, but after clicking the checkmark, do you at least see the outgoing arrow icon on the pop up box? If so, this is the icon to visit the link you've saved, but as I've mentioned, I'll bring up the fact that the URL itself does not appear 🙏

![image](https://github.com/standardnotes/forum/assets/54989748/2efe5eef-e3d0-444a-85cd-3b6fc6635b33)

**Reply by DarkArc** [reply]
This 100% is not just a visual bug.

- That button doesn't do anything.
- You can leave the note and come back and...
  - the button still doesn't do anything.
  - there's still no URL text.
- There's no note saving indicator when you do the edit.

**Reply by effieeee** [reply]
Hmm, that's odd, because the indicated URL should already be displayed on the box in the current versions. Upon testing on the demo app as well, both the button and the actual URL text redirect to the page indicated.

![image](https://github.com/standardnotes/forum/assets/54989748/db6ca453-6063-4193-b4cd-23596539abfd)

Can I ask you to check on the demo app as well? If you find that the issue appears to be specific to your account or certain notes, can I ask you to send an email to help@standardnotes.com using your account email?

**Reply by DarkArc** [reply]
The issue seems to have been fixed at some point in the last 4 days, probably by this https://github.com/standardnotes/app/commit/b9e6f66f422e4b7180e58ff748c8e9f0c235ca66
